symphony/bdk/gen/rest.py

```diff
@@ -172,9 +172,6 @@ async def request(self, method, url, query_params=None, headers=None,
             data = await r.read()
             r = RESTResponse(r, data)
 
-            # log response body
```

Comment:
Please update mustache templates  in templates/

Reply:
updated in next commit. logger is removed as well as it is not used anymore.